File: RSPAClient/Core/AtPar.ViewModel/VM_POU_PHYSICIAN_USAGE_DETAILS.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AtPar.ViewModel
{
    public class VM_POU_PHYSICIAN_USAGE_DETAILS
    {
        public string PHYSICIAN_ID { get; set; }
        public string EXAM_ID { get; set; }
        public string PREF_LIST_ID { get; set; }
        public string CASE_ID { get; set; }
        public string ITEM_ID { get; set; }
        public string ITEM_DESCRIPTION { get; set; }
        public double? ITEM_COUNT { get; set; }
        public double? USED_QTY { get; set; }
        public double? WASTAGE_QTY { get; set; }
        public string UOM { get; set; }
        public string ISSUE_UOM { get; set; }
        public double? ISSUE_PRICE { get; set; }
        public double? SUMUSED { get; set; }
        public double? SUMRETURN { get; set; }
        public double? SUMWASTAGE { get; set; }
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/VM_POU_PHYSICIAN_USAGE_DETAILS.ts
export class VM_POU_PHYSICIAN_USAGE_DETAILS {
    public PHYSICIAN_ID: string;
    public PHYSICIAN_NAME: string;
    public PREF_LIST_ID: string;
    public EXAM_ID: string;
    public CASE_ID: string;
    public ITEM_ID: string;
    public ITEM_DESCRIPTION: string;
    public UOM: string;
    public ITEM_COUNT: any;
    public USED_QTY: any;
    public WASTAGE_QTY: any;
    public ISSUE_UOM: string;
    public ISSUE_PRICE: number;
    public SUMUSED: any;
    public SUMRETURN: any;
    public SUMWASTAGE: any;
    public HeaderList: any = [];
    public HeaderColumnList: any = [];
    public DataList: any = [];
    public Details: any = [];
    public DetailsTotals: any = [];
    public PROCEDURE_CODE: string;
    public RETURNQTY: string;


}